### DISCUSSION TITLE
https://ais-web.test.dp.tech/datasets/detail?pageType=datasets&id=286

### DISCUSSION THREAD

**Discussion by giscus** [yanyumei/giscus-test#16] [Announcements]
# https://ais-web.test.dp.tech/datasets/detail?pageType=datasets&id=286

https://ais-web.test.dp.tech/datasets/detail?pageType=datasets&id=286

**Comment by Mile-Away** [comment]
这是 VDVD Version 2 的评论区，他们的评论区不互通

**Reply by Mile-Away** [reply]
而且我发现发布新版本时名字可以变化。

**Reply by Mile-Away** [reply]
发布新版本时名字不应该变化。
